app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyChoiceQuestionScreen.kt

```diff
@@ -0,0 +1,108 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.foundation.BorderStroke
```

Comment:
Not sure if this setting is updted on your AS.
<img width="983" alt="image" src="https://user-images.githubusercontent.com/8093908/196094694-cb951f8a-1b57-4f3b-86da-fa7bba384e31.png">
<img width="437" alt="image" src="https://user-images.githubusercontent.com/8093908/196094712-f632491d-5e3a-4dcf-bda5-89674de7a2b7.png">